releasenotes/notes/windowssites-0b52e956621bef1d.yaml

```diff
@@ -0,0 +1,18 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Added new command line properties to the Windows installer which allow for setting site specific configuration.
+    New flags are
+      DD_SITE sets the site override flag in datadog.yaml
+      DD_URL  sets the dd_url entry in datadog.yaml
+      LOGS_DD_URL sets the logs agent specific url in datadog.yaml
+      PROCESS_DD_URL sets the process agent specific url in datadog.yaml
+      TRACE_DD_URL sets the trace agent specific url in datadog.yaml
```

Comment:
nit: I don't think we need to list them all in details in the release note, since it's not a breaking change or anything users _have to_ understand in details before they upgrade. Details on what each option does could be in the docs only IMO